Formula/afflib.rb

```diff
@@ -15,25 +15,33 @@ class Afflib < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
-  depends_on "expat" => :optional
-  depends_on :osxfuse => :optional
+
   depends_on "openssl"
 
+  depends_on "python" => :optional
+  depends_on :osxfuse => :optional
+
   # This patch fixes a bug reported upstream over there
   # https://github.com/simsong/AFFLIBv3/issues/4
   patch :DATA
```

Comment:
Would you consider sending this patch as a PR (as requested on that now-closed issue)? Thanks!